src/components/MapSelectors/datasets/us_states_dataset_01_02_2020.json

```diff
@@ -9359,6 +9359,30 @@
         }
       ]
     },
+    "DC": {
+      "dataset": [
+        {
+          "counties": [
+            "District of Columbia"
+          ],
+          "population": 705749
+        }
+      ],
+      "county_dataset": [
+        {
+          "county": "District of Columbia",
+          "county_url_name": "district_of_columbia",
+          "county_fips_code": "001",
+          "state_fips_code": "11",
+          "state_code": "DC",
+          "full_fips_code": "11001",
+          "cities": [
+            "Washington"
+          ],
+          "population": 705749
+        }
+      ]
+    },
```

Comment:
So I added DC (as a county) to us_states_dataset because it was missing which caused a problem because our API outputs it as a county (in addition to outputting it as a state).